analyzers/backend/docker/build.go

```diff
@@ -38,16 +38,32 @@ func (d *DockerBuildError) Error() string {
 	return d.Message
 }
 
-// BuildAnalyzerDockerImage is the docker image build API used by various CLI commands
-func (d *DockerClient) BuildAnalyzerDockerImage() (context.CancelFunc, io.ReadCloser, *DockerBuildError) {
+// SetupClient initializes the Docker client with opts.
+func (d *DockerClient) SetupClient() error {
```

Comment:
Please make the changes corresponding to this function in the `verify` command as well.

Reply:
Resolved in https://github.com/deepsourcelabs/cli/pull/161/commits/b2644ca59114637f95c7c0c336f345648bc0efc5.